validation/standard/standard-versions.toml

```diff
@@ -0,0 +1,68 @@
+["op-contracts/v1.4.0-rc.3"]
+optimism_mintable_erc20_factory = "1.9.0"
+optimism_portal = "3.8.0"
+system_config = "2.2.0"
+l1_erc721_bridge = "2.1.0"
+l1_standard_bridge = "2.1.0"
+address_manager = ""
+l1_cross_domain_messenger = "2.3.0"
+protocol_versions = "1.0.0"
+anchor_state_registry = "1.0.0"
```

Comment:
We should add in `FaultDisputeGame` and `PermissionedDisputeGame` to this list.

They are not proxied.

Reply:
[afcbe66](https://github.com/ethereum-optimism/superchain-registry/pull/297/commits/afcbe66d9d61f72ca5f552d50f0f546392bca8aa)